src/stages.js

```diff
@@ -0,0 +1,121 @@
+import exec from 'k6/execution'
+
+// parseDuration parses the provided string as an Integer number
+// in millisecond precision
+function parseDuration(str) {
```

Comment:
I'm wondering, should we maybe also detect somehow negative cases :thinking: 

I mean from what I see (please correct me if I am wrong), but currently:

```js
parseDuration("0") === parseDuration("")
```